swan-lake/integration-tools/edi-tool.md

```diff
@@ -61,339 +61,245 @@ The command options that are available with the tool are listed below.
+
+- The `schemas` folder: includes the schema file ([`schemas/edi-schema.json`](https://github.com/ballerina-guides/integration-samples/tree/main/edi-code-generation/schemas/edi-schema.json)), which can be used to parse EDI documents with one HDR segment (mapped to `header`) and any number of ITM segments (mapped to `items`). The HDR segment contains three fields, which are mapped to `orderId`, `organization`, and `date`. Each ITM segment contains two fields mapped to `item` and `quantity`. 
+- The [`main.bal`](https://github.com/ballerina-guides/integration-samples/tree/main/edi-code-generation/main.bal) file: includes the business logic/usage of the code.
+- The [`resources/edi-sample.edi`](https://github.com/ballerina-guides/integration-samples/tree/main/edi-code-generation/resources/edi-sample.edi) file: includes the EDI message to be read from the generated code.
+
+#### Generate the code
+
+Follow the steps below to generate Ballerina records for the above EDI schema.
+
+1. Navigate to the cloned `edi-code-generation` directory.
+
+2. Run the tool with the [required arguments](#code-generation-command-options) to generate the package.
 
     ```
-    $ bal edi codegen -s resources/edi-schema.json -o modules/hmartOrder/orderRecords.bal
+    $ bal edi codegen -s schemas/edi-schema.json -o orderRecords.bal
+    Generating code for schemas/edi-schema.json...
     ```
 
-    The Ballerina records generated for the above schema in the `orderRecords.bal` file are shown below.
+    >**Info:** The Ballerina records below will be generated and saved in a file named `orderRecords.bal`. 
 
     ```ballerina
-    type Header_Type record {|
-        string code?;
+    public type Header_Type record {|
+        string code = "HDR";
         string orderId?;
         string organization?;
         string date?;
     |};
 
-    type Items_Type record {|
-        string code?;
+    public type Items_Type record {|
+        string code = "ITM";
         string item?;
-        int quantity?;
+        int? quantity?;
     |};
 
-    type SimpleOrder record {|
-        Header_Type header;
-        Items_Type[] items?;
+    public type SimpleOrder record {|
+        Header_Type? header?;
+        Items_Type[] items = [];
     |};
     ```
 
-3. Use the generated `fromEdiString` function to read EDI text files into the generated Ballerina record, as shown below. 
+#### Use the generated code
 
-    >**Note:** Any data item in the EDI can be accessed using the record's fields, as shown in the example code below.
+Follow the steps below to use the generated `fromEdiString` function to read EDI text files from the generated Ballerina records.
 
-    ```ballerina
-    import ballerina/io;
+##### Read from EDI files
+
+1. Navigate to the cloned `edi-code-generation` directory.
+
+2. Execute the command below to run the `read_edi()` function.
+
+    >**Note:** Any data item in the EDI text (`edi-sample.edi` file) can be accessed using the record's fields.
```

Comment:
What is the diff between Note and Info blocks?

Reply:
`Note` is used for things that require special attention from the readers such as mandatory things that should not be missed. `Info` is used mainly to give additional information about what is being discussed.

Changed to `info` as it seems to be more applicable in this particular occurrence.